rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -186,4 +186,12 @@ enum class RocalROICordsType {
 typedef struct {
     unsigned x1, y1, x2, y2;
 } RocalROICords;
+
+enum RocalAudioBorderType
+{
```

Comment:
Please move the braces to previous line to follow google style

Reply:
done